tests/benchmarking/test_benchmarking.py

```diff
@@ -75,127 +74,7 @@ def test_top_level_large_graph(large_graph, benchmark):
             containers=("mypackage",),
         )
     )
-    assert result == {
```

Comment:
♻️ https://github.com/seddonym/grimp/pull/177#discussion_r1921050223

Reply:
I'd rather we didn't remove these - we don't actually assert on the exact results of a large graph like this anywhere else and it does give me more confidence that things are happening correctly. Happy for it to be moved out of benchmarks if you feel strongly, but maybe we should leave that for a separate PR.

Reply:
Okay - put this back